### ISSUE TITLE
Add photos and create layout sections/blocks on the 175 site then/now page 

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#4758]
Add photos (link below) and create layout sections/blocks on the 175 site then/now page: https://175.uiowa.edu/then-and-now.

Using the first two sections on https://175.uiowa.edu/then-and-now as an example, build out the remaining sections with the then and now photos. 

https://iowa.sharepoint.com/:f:/s/osc/pvs/Eh-juQq_S2ZFuu8aGQF5I3kBpmbnlFwV4Xa19ZcSlipFoQ

**Comment by mjoneill** [comment]
Thanks to everyone who has touched this so far (on the team and off).  

Jeneane really likes what she is seeing.  The next step is to swap lorem ipsum copy for our best take on it.  Is this something @ABoston88 can pick up?  Once we get this to potentially shippable, we can turn it back to Jeneane and they can edit/revise as necessary.

**Comment by ABoston88** [comment]
I can take a look. Do we have any info on the context of the photos? I can do some digging, if not. Would help make the copy more specific.

**Comment by mjoneill** [comment]
> Do we have any info on the context of the photos?

Yep...it should all be in the image exif info/metadata.  If it's not there, we can help sort through it.  Kirk has it.

**Comment by ABoston88** [comment]
Draft content is in place. I learned enough from meta-data to stay broad but if we need more, let me know.

**Comment by mjoneill** [comment]
@ABoston88 thx!  I'll take a look. 

Aside: I just pushed out the 175 article.

**Comment by mjoneill** [comment]
@ABoston88 

General: Not what I was expecting, but I really like the path you took with this.  Nicely done!

Can you check the padding/space in between section heading and image in:
- 1896 Section
- 1915 Section

> even when our methods were a bit less sophisticated.

Still chewing on this.  Feels like we wouldn't want to own "we weren't sophisticated".

> 1939 section

The copy would suggest that one of the photos was about med and one was about hydroscience.  Feels like the copy doesn't quite connect in the way some of your other copy does.

**Comment by ABoston88** [comment]
Copy changed. I think the spacing change between text and image might be because of the background pattern. Seems to only be happening in those sections.

**Comment by mjoneill** [comment]
JB reached out this weekend and asked when this would be ready.  I published the page so she could see it and share the link with her stakeholders + invited feedback on copy, etc.

I also shared that:

- This is published, but not promoted on the site
- If the URL changes, we will add redirects (she is likely sharing the URL now to get feedback)
- [ ] We have yet to add it to site navigation
- [ ] We have yet to promote it on the site front page (if we would do that)
- [ ] We have yet to add a news feature for the page

Tagging @ABoston88 and @bspeare for visibility.
